dask_ml/decomposition/pca.py

```diff
@@ -190,11 +216,19 @@ def __init__(
         self.n_components = n_components
         self.copy = copy
         self.whiten = whiten
+        self.center = center
         self.svd_solver = svd_solver
         self.tol = tol
         self.iterated_power = iterated_power
         self.random_state = random_state
 
+        if whiten and not center:
+            warnings.warn(
+                "Whitening requires centering. Please, ensure that your data "
+                "is already centered, in order to avoid unexpected behavior.",
+                RuntimeWarning,
```

Comment:
Users should have some way to silence this warning. I'm inclined to just document this requirement, and trust that the user has done it (no warning).

Reply:
Sure, I'll remove the warning (the new param has been already added to the doc-string). Please, let me know if any other documentation-related changes are necessary. I've checked `api.rst` and `compose.rst` but it doesn't seem that we need any updates to either.